Source/WebCore/platform/graphics/FontDescription.h

```diff
@@ -46,7 +46,9 @@ class FontDescription {
 
     float computedSize() const { return m_computedSize; }
     unsigned computedPixelSize() const { return unsigned(m_computedSize + 0.5f); }
+    bool hasFontSizeAdjust() const { return m_sizeAdjust.has_value(); }
```

Comment:
Why wouldn't callers just call fontSizeAdjust() and convert to a bool context at that point?